src/netex_idf/offers.rs

```diff
@@ -218,13 +262,20 @@ fn parse_offer(offer: &Element, collections: &Collections) -> Result<CollectionW
                 NETEX_STRUCTURE
             )
         })?;
+    let map_journeypatterns = structure_frame
```

Comment:
I know this map could be directly a map:
- key: `route_ref`
- value: `journey_pattern_ref`
But I already know I'm gonna need the map this way for the trips so I'm keeping this intermediate step here and create the new map inside `enhance_with_object_code`.